cassandra/cqlengine/connection.py

```diff
@@ -28,67 +28,200 @@
 
 NOT_SET = _NOT_SET  # required for passing timeout to Session.execute
 
-Host = namedtuple('Host', ['name', 'port'])
-
-cluster = None
-session = None
-lazy_connect_args = None
-lazy_connect_lock = threading.RLock()
-
+# connections registry
+DEFAULT_CONNECTION = '_default_'
+_connections = {}
 
 # Because type models may be registered before a connection is present,
 # and because sessions may be replaced, we must register UDTs here, in order
 # to have them registered when a new session is established.
 udt_by_keyspace = defaultdict(dict)
 
 
+def format_log_context(msg, connection=None, keyspace=None):
```

Comment:
Should this be `connection_name` so I don't get confused about passing connection objects in?